### PR TITLE
Spelling / conventions and grammar fixes

### PR DESCRIPTION
**Pull request by HeroMaxPower**
I've corrected some spelling, grammar & whitespace errors i noticed while in this solution.

Please take a look. Hopefully it will improve the information conveyed to the user / dev.

### CODE REVIEW

File: test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs [hunk 1]
```diff
@@ -6,106 +6,106 @@
 
 namespace vstest.console.UnitTests.TestPlatformHelpers
 {
-    using System;
```
Comment (nohwnd) [inline test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs]: what is going on in this file? was it using Tabs as spaces?

File: test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs [hunk 2]
```diff
@@ -3,842 +3,842 @@
 
 namespace Microsoft.TestPlatform.Utilities.UnitTests
 {
-    using System;
```
Comment (nohwnd) [inline test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs]: what is going on here? were there tabs instead of spaces?

**Review by nohwnd — APPROVED** [review]

File: test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs (hunk 1 above)
Comment (nohwnd) [reply test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs]: hard to see from this view but the file has few thousand changes.

File: test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs (hunk 2 above)
Comment (nohwnd) [reply test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs]: hard to see from this view but the file has few thousand changes.

File: test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs (hunk 1 above)
Comment (HeroMaxPower) [reply test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs]: Yes so i used the [FixMixedTabs](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs) plugin to remove spaces with tabs, so looks like the plugin rewrites the whole cs file.

File: test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs (hunk 2 above)
Comment (HeroMaxPower) [reply test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs]: Yes so i used the [FixMixedTabs](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs) plugin to remove spaces with tabs, so looks like the plugin rewrites the whole cs file.

**Comment by nohwnd** [comment]
Thanks, merged 👊

**Pull request merged**